src/components/slide.js

```diff
@@ -95,11 +102,17 @@ const Slide = props => {
   return (
     <SlideContainer
       ref={slideRef}
-      backgroundColor={backgroundColor}
-      style={{
-        transform: `scale(${ratio})`,
-        transformOrigin: `${origin.x} ${origin.y}`
-      }}
+      backgroundColor={
+        window.location.search.includes('print') ? '#ffffff' : backgroundColor
+      }
+      style={
+        state.exportMode
+          ? {}
+          : {
+              transform: `scale(${ratio})`,
+              transformOrigin: `${origin.x} ${origin.y}`
+            }
```

Comment:
This is starting to get a little big, perhaps time to move to a React.useMemo?

Reply:
Sure can do!